src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
```

Comment:
an empty line between methods declarations is not redundant, let's return this one back

some of the empty lines are necessary and your build failed without them, run mvn clean package locally to see what's wrong and fix it